src/main/java/org/myrobotlab/service/AdafruitMotorHat4Pi.java

```diff
@@ -147,6 +147,7 @@ public AdafruitMotorHat4Pi(String n, String id) {
     super(n, id);
     refreshControllers();
     subscribeToRuntime("registered");
+    registerForInterfaceChange(I2CController.class);

```

Comment:
the framework should call this, subclasses shouldn't have to worry about this detail here.